OneDriveSDK/OneDriveCoreSDK/requests/ODDriveRequestBuilder.m

```diff
@@ -70,4 +75,9 @@ - (ODDriveRequest *) requestWithOptions:(NSArray *)options
     return [[ODDriveRequest alloc] initWithURL:self.requestURL options:options client:self.client];
 }
 
+- (ODDriveRecentRequestBuilder *)recent

```

Comment:
FYI - You are including view.recent, might want to quickly gut check this functionality.